torch/_ops.py

```diff
@@ -40,8 +47,60 @@ def py_impl(self, dispatch_key, fn):
     def name(self):
         pass
 
-    def dispatch(self, dispatch_key, *args, **kwargs):
-        pass
+
+is_included_in_alias = torch._C._dispatch_is_included_in_alias
+
+DispatchKey = torch._C.DispatchKey
+
+# Equivalent to computeDispatchTableEntryWithDebug
+def resolve_key(op: PyOperatorABC, k: DispatchKey):  # type: ignore[valid-type]
```

Comment:
This probably isn't going to break anything in the near future, but we'll have to remember to keep these in sync:

(1) `[CompositeImplicitAutogradNestedTensor](https://github.com/pytorch/pytorch/blob/f37069aac7a0c4fd1c3455e4e9058bf56fb759f4/aten/src/ATen/core/dispatch/OperatorEntry.cpp#L330)` is missing

(2) Note to myself to add `Autocast` here if we end up making autocast an [alias key](https://github.com/pytorch/pytorch/pull/85043/files#diff-da6c1aa633224c76f5ae6b4e80676dadbb24bac021770ee01d5560d2db8ffb92R88)

Reply:
It's here? 

```
    # 2.3. Use CompositeImplicitAutograd kernel if available
    cand = DispatchKey.CompositeImplicitAutogradNestedTensor
    if (
        (k != DispatchKey.Undefined and is_included_in_alias(k, cand))
        and has_key(op, cand)
        and not has_backend_kernel
    ):
        return cand
    cand = DispatchKey.CompositeImplicitAutograd
    if (k == DispatchKey.Undefined or is_included_in_alias(k, cand)) and has_key(
        op, cand
```

Reply:
oh whoops haha